src/cloud-api-adaptor/test/e2e/main_test.go

```diff
@@ -97,10 +97,14 @@ func TestMain(m *testing.M) {
 
 	// The DEPLOY_KBS is exported then provisioner will install kbs before installing CAA
 	shouldDeployKbs := false
-	if os.Getenv("DEPLOY_KBS") == "true" || os.Getenv("DEPLOY_KBS") == "yes" {
+	if isTestWithKbs() {
 		shouldDeployKbs = true
 	}
```

Comment:
Could we change this to `shouldDeployKbs := isTestWithKbs()`, or is there something about the negative default I'm missing?

Reply:
oh, silly mistake :-(